modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -583,6 +588,9 @@ public void clear() {
     public void addThrottlingTime(long time) {
         if (metricsEnabled)
             totalThrottlingTime.add(time);
+
+        if (ctx.performanceStatistics() != null && ctx.performanceStatistics().enabled())
```

Comment:
Why `performanceStatistics` can be null here?

Reply:
fixed